tests/manager/manager-ctor-write_concern-005.phpt

```diff
@@ -0,0 +1,74 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): write concern options (64-bit wtimeoutms)
+--SKIPIF--
+<?php if (4 !== PHP_INT_SIZE) { die('skip Only for 32-bit platform'); } ?>
```

Comment:
Why would this test only run on 32-bit platforms? I don't see anything that would prohibit a 64-bit platform from passing it, given that the connection string parsing logic in libmongoc is the same. If anything, I'd think that a 32-bit platform wouldn't be able to deal with `2147483648` integer literals, as you have below for the URI option array test cases.

Reply:
Test simplified to cover 32-bit and 64-bit systems using `EXPECTF`.